frontend_app_file/README.md

```diff
@@ -1,18 +1,37 @@
-# App File for Tracim
-
-This repo is an app loaded by Tracim.
-
-### To update this repo
-- commit and push changes
-- run `$ npm run build`
-- copy dist/pageHtml.app.js and past it into Tracim(repo)/dist/app
-
-### Development
-To see your changes without importing the app into Tracim you need to be logged in to the frontend's dev server to have an auth cookie an run:
-- run `$ npm run servdev`
-Which will create a web server (webpack) where you will see the app loaded with some default values.
-For this command the configuration file debug.js is used which follows the template of the file debug.sample.js. To adapt the configuration to your content, you need to change the variables:
- - content_id: integer
- - workspace_id: integer
- - apiUrl: /api (this can be a full URL)
-Another needed configuration is made in the backend/development.ini, where you need to add http://MACHINE_ADRESS:8075 in to the variable cors.access-control-allowed-origin
```

Comment:
I think this part has not been transferred to any document.

Reply:
I now link to the same documentation file frontend/doc/app_building#Development in the "#### For development" part
it avoid repeating the same thing in each app's readme

Reply:
but there is nothing about cors.access-control-allowed-origin at app_building file.. maybe it is not that important?

Reply:
oh, I miss read your comment
you are right this part is missing.
I will add it to frontend/doc/app_building.md

Reply:
will fix

Reply:
fixed